src/main/java/com/igot/cb/competencies/subtheme/service/impl/CompetencySubThemeServiceImpl.java

```diff
@@ -545,8 +545,8 @@ public ApiResponse createTerm(JsonNode request) {
               response.setResponseCode(HttpStatus.INTERNAL_SERVER_ERROR);
               response.getParams().setStatus(Constants.FAILED);
             }
-          } else if (HttpStatus.OK.equals(readResponse.getResponseCode())) {
-            String errMsg = "Term already exists with name: " + ref_Id;
+          } else if (HttpStatus.CONFLICT.equals(readResponse.getResponseCode())) {
```

Comment:
What is this status type and from where it will fire

Reply:
We are setting  this status in the read Framework method and using this status for checking the duplicate and returing the approapriate error message to FE